touchforms/formplayer/static/formplayer/script/fullform-ui.js

```diff
@@ -530,6 +593,20 @@ Formplayer.Utils.initialRender = function(formJSON, resourceMap, $div) {
         $debug.koApplyBindings(cloudCareDebugger);
     }
 
+    if ($inFormNav.length) {
+        webformsNavigator = new Formplayer.ViewModels.WebformsNavigator(form);
+        ko.cleanNode($inFormNav[0]);
+        $inFormNav.koApplyBindings(webformsNavigator);
+
+    }
+
     return form;
 };
 
+FormplayerFrontend.on("webforms:clearForm", function () {
```

Comment:
I'm getting a js error locally from this line that seems to be breaking cloudcare. It doesn't look like this PR is going to be deployed today (don't see a submodule commit) but would probably be good to take a look.

![screen shot 2016-08-31 at 1 56 34 pm](https://cloud.githubusercontent.com/assets/1486591/18140268/c0d9429e-6f82-11e6-8f46-0093ce41d384.png)